src/ontology/mondo-ingest.Makefile

```diff
@@ -181,6 +182,8 @@ config/%_term_exclusions.txt: config/%_exclusions.tsv component-download-%.owl $
 	--component-signature-path $(word 4,$^) \
 	--config-path $(word 5,$^)
 
+all-exclusions: config/ordo_term_exclusions.txt config/doid_term_exclusions.txt config/omim_term_exclusions.txt config/icd10cm_term_exclusions.txt config/icd10who_term_exclusions.txt config/ncit_term_exclusions.txt
```

Comment:
@matentzn I'm almost done running for all ontologies, the exception being `icd10cm` which will need that empty `.owl` file fix.

Reply:
its fixed! #60

Reply:
nice